apps/website/foundation/app-layout/README.md

```diff
@@ -3,6 +3,8 @@ title: Overview
 toc: true
 ---
 
+<DocAlert status="warning" path="/storybook/core/?path=/story/layout-get-started--page">This content is for the <b>@clr/angular</b> package. Layout information for Core Components can be found in storybook.</DocAlert>
```

Comment:
Can we link to the Storybook URLs?

Reply:
Yes but I think I should probably open a new tab here since its a different app.